src/components/calculator.js

```diff
@@ -0,0 +1,70 @@
+const Calculator = () => (
+  <div className="calculator">
+    <div className="screen ">0</div>
+    <div className="keyboad">
+      <button type="button" className="numpad">
+        AC
+      </button>
+      <button type="button" className="numpad">
+        +/-
+      </button>
+      <button type="button" className="numpad">
+        %
+      </button>
+      <button type="button" className="numpad operator">
+        +
+      </button>
+
+      <button type="button" className="numpad">
+        7
+      </button>
+      <button type="button" className="numpad">
+        8
+      </button>
+      <button type="button" className="numpad">
+        9
+      </button>
+      <button type="button" className="numpad operator">
+        x
+      </button>
+
+      <button type="button" className="numpad">
+        4
+      </button>
+      <button type="button" className="numpad">
+        5
+      </button>
+      <button type="button" className="numpad">
+        6
+      </button>
+      <button type="button" className="numpad operator">
+        -
+      </button>
+
+      <button type="button" className="numpad">
+        1
+      </button>
+      <button type="button" className="numpad">
+        2
+      </button>
+      <button type="button" className="numpad">
+        3
+      </button>
+      <button type="button" className="numpad operator">
+        ÷
+      </button>
+
+      <button type="button" className="numpad zero">
+        0
+      </button>
+      <button type="button" className="numpad">
+        .
+      </button>
+      <button type="button" className="numpad operator">
+        =
+      </button>
+    </div>
+  </div>
+);
+
```

Comment:
- [ ] According to the previous reviewer, inside your components folder you should create `Calculator.js` file but I notice that your name is `calculator.js`.